corehq/apps/translations/app_translations.py

```diff
@@ -725,25 +725,20 @@ def update_form_translations(sheet, rows, missing_cols, app):
         return msgs
 
     def find_itext_node(xform):
-        try:
-            # find the bucket node that holds all translations.
-            # it has a bunch of nodes, one for each lang, which then
-            # has translations for all labels as a child node, example
-            # <itext>
-            #   <translation lang="en" default="">
-            #     <text id="name-label">
-            #       <value>Name2</value>
-            #       <value form="image">image_path</value>
-            #     </text>
-            #   </translation>
-            # </itext>
-            return xform.itext_node
-        except XFormException:
-            return None
```

Comment:
the exception is raised [if there is no itext node](https://github.com/dimagi/commcare-hq/blob/0a9970e23606d62488069d01bb517815c00a085c/corehq/apps/app_manager/xform.py#L653) and i don't think that would ever be the case.

Reply:
But if you think there is a case i am missing like if a blank form is added and that has no questions added to it then this would raise exception, i am happy to add this back but from what i seemed to me this exception would never be raised.